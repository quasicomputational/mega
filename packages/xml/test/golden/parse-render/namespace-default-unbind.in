<foo xmlns="a"><bar xmlns="" /></foo>