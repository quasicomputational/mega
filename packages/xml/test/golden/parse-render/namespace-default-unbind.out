<n1:foo xmlns:n1="a"><bar></bar></n1:foo>